### ISSUE TITLE
[Feature]: enable border and corner of controls upper  canvas.clipPath

### ISSUE THREAD

**Issue by FANGLANYAN** [fabricjs/fabric.js#10304]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

Hi!  I am trying to use Fabric to implement a canvas function. After drawing the image onto Canvas, I created a clipath area that is the same size as the image, and then moved the image. However, when my image was moved to the bottom of the clipath, the controls (border and corners) of the image were obscured by the clipath.
![image](https://github.com/user-attachments/assets/d01a2b9d-ca0b-4c66-8e8a-729ec5ab935f)
 Is there any way to prevent it from being obscured? It's like this
![image](https://github.com/user-attachments/assets/8488b77c-9c17-40a5-992a-29d6143cfe2e)
Thanks

### Current State

version:5.2.1
I also thought of finding any solution, but after browsing the community, the conclusion seems to be unsupported

**Comment by asturur** [comment]
Did you try to use controlsAboveOverlay to see if it makes any difference?
Also did you try to use an overlay instead of a clipPath to obtain your effect?

**Comment by FANGLANYAN** [comment]
Thank you very much for your reply. After carefully checking my code, I realized that I had set the controlsAboveOverlay property incorrectly🤣, applying it to my image object. After adjusting it, I was able to achieve the desired effect. Fabric is truly an excellent canvas solution that has helped me implement many cool effects.

Thank you for your patient explanations and assistance. Best regards!
